engine/src/main/db/update/scriptServ.sql

```diff
@@ -1 +1,4 @@
-
+ALTER TABLE authusers ALTER COLUMN username TYPE character varying(80);
```

Comment:
Was this intended? I can see these line were addressing a different sql script before (port) and now are in serv

Reply:
Doesn't look right to me either? Is this the right file for this.